caas/kubernetes/provider/application/application.go

```diff
@@ -1089,6 +1090,40 @@ func (a *app) Service() (*caas.Service, error) {
 	}, nil
 }
 
+func (a *app) computeStatus(ctx context.Context, client kubernetes.Interface, now time.Time) (string, status.Status, time.Time, error) {
+	jujuStatus := status.Waiting
```

Comment:
allocating is what we'd want here.
waiting is usually reserved as being for the charm to set